av_imu_launch/config/av_gx5_15.yml

```diff
@@ -1,7 +1,13 @@
-microstrain_inertial_driver:
+# Standalone example params file for GX3, GX4, GX/CX5, RQ1 and GQ7 series devices
+# Note: Feature support is device-dependent and some of the following settings may have no affect on your device.
+# Please consult your device's documentation for supported features
+
+/**:
   ros__parameters:
-    # You should change this section of config to match your setup
-    port : '/dev/imu-front'
+    # ******************************************************************
+    # University  of Edinburgh Vehicle IMU Mandatory Settings
+    # ******************************************************************
+    port : "/dev/imu-6"
```

Comment:
We could always change this to be `/dev/imu-front` in the car .rules configuration and leave this to point to imu-front (After all, we don't have so many imus in our fleet of 1 car that we need to track them by number... It should be easier to track them by their name not their number. I'll leave the decision to you)

Reply:
I like the idea, will change it to `/dev/imu-front`

Reply:
Done, we just need to remember that the udev rule or symbolic link needs to be updated